.github/workflows/helm-multi-release.yaml

```diff
@@ -0,0 +1,183 @@
+name: Helm Multi Release
+# Reusable workflow for packaging and publishing Helm charts having multiple subcharts on GitHub pages
+
+on:
+  workflow_call:
+    inputs:
+      charts-dir:
+        description: "The directory containing the Helm chart and `.bumpversion.cfg` file."
+        required: true
+        type: string
+      subdirs:
+        description: "List of subdir to consider" # "['subdir1', 'subdir2', 'subdir3']"
+        required: true
+        type: string
+      artifact-dir:
+        description: "Directory inside `charts-dir` for preparation of the GitHub pages artifact."
+        required: false
+        default: "artifacts"
+        type: string
+      gh-pages-branch:
+        description: "Name of branch containing the artifacts"
+        required: false
+        default: "gh-pages"
+        type: string
+      project-root-dir:
+        description: "Github dir where the charts dir is located"
+        required: false
+        default: "."
+        type: string
```

Comment:
What do you think about simplifying the `project-root-dir` and the `charts-dir` into one input variable by allowing the user to set a path, e.g. `./my-projects/charts`?